### PR TITLE
Added Player.java with its respective test

### PR DESCRIPTION
**Pull request by JhordanSoto**
Added Player.java with its respective test

### CODE REVIEW

File: src/test/java/chess/PlayerTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    }
+    @Test
+    public void testShowSurrender() {
+        Player player = new Player(Color.BLACK, true,"Pepe");
+        String expected = "The player Pepe surrender";
+        String actual = player.showSurrender();
+        assertEquals(expected, actual);
+    }
+
+}
```
Comment (roht0515) [inline src/test/java/chess/PlayerTest.java:40]: add a new line after "}" please

File: src/main/java/chess/Color.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package chess;
+
+public enum Color {
+    WHITE, BLACK
+}
```
Comment (roht0515) [inline src/main/java/chess/Color.java:5]: add a new line at the end

**Comment by samuelloza** [comment]
Please add to .gitignore these files build.gradle and settings.gradle.

**Comment by MirkoFer-622** [comment]
Observations were solved, it's ready to commit!

**Pull request merged**